portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,16 @@
   </head>
   <body>
     <div id="content">
-      <h1>Aymar Portfolio</h1>
+      <h1 id="header">Aymar Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>It includes my training work for the STEP internship:</p>
+      <div id="w2">
```

Comment:
Please use a more descriptive `id`

Reply:
The division no longer exists. Future id will be more descriptive

Reply:
That `div` is removed in a follow-up PR?